ci/build_container/build_recipes/benchmark.sh

```diff
@@ -6,13 +6,23 @@ export COMMIT="e1c3a83b8197cf02e794f61228461c27d4e78cfb"  # benchmark @ Jan 11,
 git clone https://github.com/google/benchmark.git
 (cd benchmark; git reset --hard "$COMMIT")
 mkdir build
-
 cd build
-cmake -G "Unix Makefiles" ../benchmark \
+
+cmake_generator="Unix Makefiles"
+make_cmd=make
+benchmark_lib="libbenchmark.a"
+
+if [[ "${OS}" == "Windows_NT" ]]; then
+  cmake_generator="Ninja"
+  make_cmd=ninja
```

Comment:
Why Ninja? Is there no way to do GNU make on Win?

Reply:
We decided to use Ninja because that is what Microsoft uses in their VCPKG tool for managing/building C/C++ libraries. For benchmark specifically, see: https://github.com/Microsoft/vcpkg/blob/d977ac231e995250c169ac1778b7de34f7f57ead/ports/benchmark/portfile.cmake#L22

and https://github.com/Microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_configure_cmake.md#prefer_ninja

Reply:
Out of curiosity, does it fail if you use GNU make? I'm curious what the Bazel folks who are working on external dependency integration with cmake plan on Windows, CC @iirina.

Reply:
For benchmark at least, we were not able to get it to work with GNU make